packages/react-components/react-storybook-addon/package.json

```diff
@@ -32,6 +32,7 @@
   "dependencies": {
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31794&commitId=e86ee0363597bf1ba2de33054d4948354f060d0f&env=prod&clientType=FLUENTUI)


  <details><summary><b>Image Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Image Converged.Image Variations Fallback.normal.chromium.png|0| Removed|

</details>


  <details><summary><b>SwatchPicker Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SwatchPicker Converged.default - RTL.default.chromium.png|0| Added|
|SwatchPicker Converged.default.default.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>